### PR TITLE
refactor: fix multiple approvals

### PR DESCRIPTION
**Pull request by viestat**
This Pull Request fulfills the following requirements:

- [x] The commit message follows our guidelines.
- [x] Tests for the changes have been added if needed.
- [x] Does not affect documentation or it has been added or updated.
- [x] Does not introduce cycles into the flow of execution.

Fixes #23

### CODE REVIEW

**Comment by viestat** [comment]
To anyone reviewing. Remember to ignore all the files in the `dist` folder. They are generated by ncc

File: src/graphql/queries.ts [hunk 1]
```diff
@@ -19,3 +19,69 @@ export const findPullRequestNodeIdByHeadReferenceName = `
     }
   }
 `;
+export const findPullRequestInfoAndReviews = `
```
Comment (merlinnot) [inline src/graphql/queries.ts]: Newline before?

File: src/util/index.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import {
+  approveAndMergePullRequestMutation,
+  mergePullRequestMutation,
+} from '../graphql/mutations';
+/**
```
Comment (merlinnot) [inline src/util/index.ts]: Newline?

File: src/util/index.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ * This method returns the right GraphQl mutation depending on weather the
```
Comment (merlinnot) [inline src/util/index.ts]: I think we can simplify it and align with other repositories.

```suggestion
 * Returns the right GraphQl mutation depending on weather the
```

File: test/TestEnv.js [hunk 4]
```diff
@@ -1,17 +1,51 @@
...
+    switch (context.docblockPragmas['webhook-pragma']) {
+      case 'check_suite':
+        env.GITHUB_EVENT_PATH = './test/ctx.check-suite.json';
+        env.GITHUB_EVENT_NAME = 'check_suite';
+        env.GITHUB_SHA = 'ffac537e6cbbf934b08745a378932722df287a53';
+        env.GITHUB_REF = 'refs/heads/test-branch';
+        env.GITHUB_WORKFLOW = 'Auto merge';
+        env.GITHUB_ACTION = 'ridedottauto-merge-action';
+        env.GITHUB_ACTOR = 'dependabot-preview[bot]';
+        env.GITHUB_REPOSITORY = 'test-actor/Test-Repo';
```
Comment (merlinnot) [inline test/TestEnv.js]: Should we sort all of these for easier navigation?

File: test/ctx.check-suite.json [hunk 5]
```diff
@@ -0,0 +1,233 @@
+{
```
Comment (merlinnot) [inline test/ctx.check-suite.json]: Should we put it in `fixtures` directory?

**Pull request merged**